torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -655,7 +655,7 @@ def view_avoid_dupes_with_primals(t):
                     t, lambda _, inner_t: view_avoid_dupes_with_primals(inner_t)
                 )
             if isinstance(t, Tensor):
-                return t.view(t.shape)
+                return t.detach()
```

Comment:
just curious, why does `.view(t.shape)` not work anymore?

Reply:
See my other comment below - I guess it shouldn't matter which one we use as long as we have a place to interpose the logic to update the registry.